src/batched-json.js

```diff
@@ -67,7 +67,19 @@ export function batchFetchJsonFor(
       }
       return xhr.fetchJson(data.xhrUrl, data.fetchOpt);
     })
-    .then(res => res.json())
+    .then(res => {
+      // https://github.com/ampproject/amphtml/issues/17691
+      const stripPrefix = element.getAttribute('strip-prefix') || '';
```

Comment:
This should really be passed as options to `batchFetchJsonFor`, giving us the change to clean up that ugly call signature:

```js
function batchFetchJsonFor(ampdoc, el, opts) {
}
```

Reply:
cleaning up the ugly call signature in https://github.com/ampproject/amphtml/pull/25505